client/lcd/test_helpers.go

```diff
@@ -1404,7 +1404,7 @@ func getVotingParam(t *testing.T, port string) gov.VotingParams {
 // ----------------------------------------------------------------------
 // GET /slashing/validators/{validatorPubKey}/signing_info Get sign info of given validator
 func getSigningInfo(t *testing.T, port string, validatorPubKey string) slashing.ValidatorSigningInfo {
-	res, body := Request(t, port, "GET", fmt.Sprintf("/slashing/validators/%s/signing_info", validatorPubKey), nil)
```

Comment:
imho, this should be reverted. The signing_info is a property of the resource which in this case is the validator.

Reply:
Just reverted